JFunc/src/main/java/functionalExamples/Example3.java

```diff
@@ -16,6 +16,7 @@ public Example3(String name, int age, String country) {
     // As this class is immutable object, so can all the get methods can be called as functions? If
     // we say they are functions but they violates the rule of function should take an input....
 
+    
```

Comment:
Example3 obj =new Example3(“pallavi”,”India”,40);

--> Here object “obj” is immutable.  
--> Get methods are not changing the state of the object.
--> Get methods are returning the values.
--> But returned value is not depending on method params. Because get method doesn't have params.
--> As per function definition, Here get methods are not functions.(As i know)